CONTRIBUTING.md

```diff
@@ -12,9 +12,9 @@ Be sure to include as much information including screenshots, text output, and b
 
 ## How to Request Enhancements
 
-First, please refer to the applicable [GitHub repository][github-repo] and search [the repository's GitHub issues][issues-list] to make sure your idea has not been (or is not still) considered.
+Firstly please refer to the applicable GitHub repository[github-repo] and search the repository's GitHub issues[issues-list] to make sure that your idea has not been (or is not still) considered.
 
-Then, please [create a new issue in the GitHub repository][new-issue] describing your enhancement.
+If it is not yet considered or something more can be added then please create a new issue in the GitHub repository[new-issue] describing your enhancement.
```

Comment:
This is not looking good to me
![image](https://user-images.githubusercontent.com/48255244/111897303-3b779000-8a45-11eb-92f4-82eac1403643.png)